packages/app/src/elements/history-entry-dialog.ts

```diff
@@ -0,0 +1,121 @@
+
+                <div class="stretch"></div>
+
+                ${fieldsChanged.includes("name") ? this._showChangedNameContent() : null}
+                ${fieldsChanged.includes("tags") ? this._showChangedTagsContent() : null}
+                ${fieldsChanged.includes("fields") ? this._showChangedFieldsContent() : null}
+
+                <div class="horizontal evenly stretching spacing layout top-margined">
+                    <pl-button class="primary" @click=${() => this._restore()}>
+                        ${$l("Restore item to this version")}
+                    </pl-button>
+                    <pl-button @click=${this.dismiss}> ${$l("Cancel")} </pl-button>
+                </div>
+            </div>
+        `;
+    }
+
+    async show(historyEntry: ItemHistory) {
+        await this.updateComplete;
+        this._historyEntry = historyEntry;
+        return super.show();
+    }
+
+    private _showChangedNameContent() {
+        return html`
+            <div class="tags border-bottom">
+                <h2 class="subtle horizontally-double-margined bottom-margined animated section-header">
+                    <pl-icon icon="text" class="inline small light"></pl-icon>
+                    ${$l("Name")}
+                </h2>
+                <div class="bottom-margined">${this._historyEntry.name || $l("<None>")}</div>
+            </div>
+        `;
+    }
+
+    private _showChangedTagsContent() {
+        return html`
+            <div class="tags border-bottom">
+                <h2 class="subtle horizontally-double-margined bottom-margined animated section-header">
+                    <pl-icon icon="tags" class="inline small light"></pl-icon>
+                    ${$l("Tags")}
+                </h2>
+                <div class="bottom-margined tiny wrapping spacing horizontal layout">
+                    ${this._historyEntry.tags.length === 0
+                        ? $l("<None>")
+                        : this._historyEntry.tags.map(
+                              (tag) =>
+                                  html` <div class="tag"><pl-icon class="inline" icon="tag"></pl-icon>${tag}</div> `
+                          )}
+                </div>
+            </div>
+        `;
+    }
+
+    private _showChangedFieldsContent() {
+        return html`
+            <div class="fields border-bottom">
+                <h2 class="subtle horizontally-double-margined bottom-margined animated section-header">
+                    <pl-icon icon="field" class="inline small light"></pl-icon>
+                    ${$l("Fields")}
+                </h2>
+                <pl-list class="border-top block">
+                    ${repeat(
+                        this._historyEntry.fields,
+                        (field) => `${field.name}_${field.type}`,
+                        (field: Field) => html`
+                            <pl-field class="padded list-item" .field=${field} .editing=${false} .auditResults=${[]}>
```

Comment:
I would set `pointer-events: none` on this to disable field actions and dragging.

Reply:
This makes it impossible to reveal/see password fields, so is there a better/different way to remove dragging that doesn't disable that important functionality?

Reply:
Good point! In that case I guess you'll either need to add something like a `readonly` property or (probably easier) simply recreate the display portion of the UI. I'd actually prefer the latter approach I think since I'd like to avoid adding more complexity to `<pl-field>`.

Reply:
I added a simple `hideEdit` to hide that button, but keep the rest as it'll be familiar UX.

Reply:
Is it still draggable though?